tensorflow/tensorboard/components/tf-event-dashboard/tf-event-dashboard.html

```diff
@@ -110,6 +110,12 @@
                       icon="fullscreen"
                       on-tap="toggleSelected"
                     ></paper-icon-button>
+                      <paper-icon-button
+                              class="log-option-button"
+                              icon="line_weight"
```

Comment:
I feel like it is confusing both ways. Maybe changing the icon background to show that the option is 'selected'  is better than changing between different icons. This way, it would become obvious if the log scale is enabled or not.